lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -202,3 +233,19 @@ def update_keyfile(self, key_file_contents: str) -> bool:
             )
 
         return True
+
+    def update_relation_data(self, relation_id: int, data: dict) -> None:
```

Comment:
We shouldn't have the already existing `DatabaseRequires.update_relation_data()` "re-exposed"

Reply:
gotcha!